admin_manual/enterprise_external_storage/windows-network-drive_configuration.rst

```diff
@@ -150,6 +150,31 @@ The ownCloud server needs to know about changes of files on integrated storages
 
 To create a new SMB notification, start a listener on your ownCloud server with ``occ wnd:listen``. The listener marks changed files, and a background job updates the file metadata.
 
+Windows network drive connections and setup of ``occ wnd:listen`` often does not
+always work the first time. If you encounter issues using it, then try the
+following troubleshooting steps:
+
+1. Check the connection with smbclient_ on the commandline of the ownCloud server
+2. If you are connecting to `Federated Cloud Shares`_, be aware that the shares are case-sensitive
```

Comment:
In this case [DFS shares](https://de.wikipedia.org/wiki/Distributed_File_System_(Microsoft)) are relevant, not Federated Cloud shares.

🤗 for the quick add!